lib/bank_accounts.rb

```diff
@@ -0,0 +1,11 @@
+module BankAccounts
+
+  # NOTE TO SELF:
+  # require_relative is AMAZING -- doesn't matter what folder you're in terminal
+  # as long as you reference this file correctly, these will work
+  # "require" is dependant on terminal location
+  require_relative './bank_accounts/account'
+  require_relative './bank_accounts/savings_account'
+  require_relative './bank_accounts/checking_account'
+
```

Comment:
Since rspec automatically does the proper file loading here, you should even be able to simply say `require `bank_accounts/account` etc. :)